### DISCUSSION TITLE
Physics informed neural operators

### DISCUSSION THREAD

**Discussion by mattragoza** [lululxvi/deepxde#927] [Ideas]
I am interested in using deepxde framework for operator learning constrained by a physical PDE. However I would like to use neural operators rather than DeepONet as I need to be able to generalize to input functions with different geometries (the "sensor points" are not consistent).

I think I need a new data format that supports quadruple data instances (x, a, y, u) which represent samples of the input function a(x) and output function u(y), and where y can be a different set of points than x. However I want to train the model using physics informed loss, like data.PDE or data.PDEOperator. So I can't just use data.Quadruple.

I would like to contribute features that would make this possible. Any suggestions on how to approach this?

**Comment by lululxvi** [comment]
Yes, sounds good. I think you can start with the implementation of pure data driven loss, and then implement the physics loss.
